layouts/partials/version-banner.html

```diff
@@ -20,11 +20,15 @@
 
   {{ $color := "warning" -}}
   <div class="alert alert-warning" role="alert">
-    <p>Version <strong>{{ $thisVersName }}</strong> of
+    <p>
+    <i class="fa fa-exclamation-triangle" aria-hidden="true">&nbsp;</i>
+    {{ if eq $thisVers "next" }}
     {{ $pageWarning | markdownify }}
-    For the latest stable documentation, see
-    <a href="{{ $targetURL }}">
-      {{- $vers -}}
-    </a>.</p>
+    {{ else }}
+    Documentation for version {{ $thisVersName }} is no longer actively maintained.
+    The version you are currently viewing is a static snapshot.
```

Comment:
I'm trying to think of a better way of doing this (this also relates to cutting `deprecation_warning` out of `config.yaml`). Keeping in mind that I'd like to keep as much copy as possible internationalization friendly, The issue I had was that I have a variable in the middle of the sentence, and if i left it in the config (as is), then Hugo renders `{{ $thisVersName }}` in the copy instead of trying to resolve it.